news/2017-07-10-osucatch-world-cup-2017-results.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "osu!catch World Cup 2017 - Results"
+permalink: http://osunews.tumblr.com/post/162821221298/osucatch-world-cup-2017-results
+date: 2017-07-10 12:30:00 +0000
+tumblr_url: http://osunews.tumblr.com/post/162821221298/osucatch-world-cup-2017-results
+---
+
+![](/wiki/shared/news/banners/catch_logo_2017.png)
+
+The past five weeks have seen 189 intrepid fruit-catchers from 32 different countries duke it out for the ultimate prize of being crowned the 2017 osu!catch World Champions. We've seen thrills and spills, crises and comebacks, as the best players in the world showcased their talent in a total of 73 exciting matches. When all was said and done, only two of those teams remained standing for a final, fateful encounter. South Korea and China, two true powerhouses of the game would take to the stage one final time, both bringing their own dreams and aspirations. China, looking to claim their second victory in a row after triumphing over Indonesia in 2016. South Korea, looking to return to the winners' circle after emerging triumphant in 2014 and 2015. Could South Korea repeat their Winners' Bracket victory, or would China return with a vengeance to steal the show? Read on to find out, or check out the video on demand below!
```

Comment:
Generally speaking, paragraphs longer than 4 lines on a full-screen 1080p diff word wrap are wayyy too long.

Split paragraph at ``South Korea and China`` and maybe consider using a variation on the last sentence of this section as the 'header' paragraph